src/renderer/components/+apps-releases/release.store.ts

```diff
@@ -82,23 +86,23 @@ export class ReleaseStore extends ItemStore<HelmRelease> {
   async create(payload: IReleaseCreatePayload) {
     const response = await helmReleasesApi.create(payload);
 
-    if (this.isLoaded) this.loadAll();
+    if (this.isLoaded) this.loadSelectedNamespaces();
```

Comment:
I suggest adding a comment that this is supposed to be a refreshing call

Reply:
Probably should be refactored to separated method